roles/ceph-osd/tasks/check_devices.yml

```diff
@@ -40,7 +40,7 @@
   always_run: true
   register: journal_partition_status
   when:
-    - (raw_multi_journal or dmcrypt_dedicated_journal)
+    - (osd_scenario == 'raw_multi_journal' or osd_scenario == 'dmcrypt_dedicated_journal')
```

Comment:
does ``(osd_scenario in ['raw_multi_journal', 'dmcrypt_dedicated_journal'])`` work here?